P001/modelsim.mk

```diff
@@ -0,0 +1,22 @@
+
+all: work
+	vcom -work work P001.vhdl
+	vcom -work work testbench.vhdl
+
+testbench.wlf: all 
+	vsim -c -do sim.do testbench -wlf testbench.wlf
+
+testbench.vcd: testbench.wlf
+	wlf2vcd -o $@ $<
+
+show: testbench.vcd
+	gtkwave testbench.vcd
```

Comment:
You could use Modelsim to display your wlf or use it graphically during the simulation.

Modelsim can also generate a Makefile (I think the command is vmake), but unless you have a really complicated group of .vhd files with dependecy, it is not worth it.